### DISCUSSION TITLE
New Playtime Reward Claim Command

### DISCUSSION THREAD

**Discussion by BboyZeroOne** [Zrips/CMI#3453] [Feature Requests]
**Description of feature/improvement request:**
Add a new command to claim all playtime reward instead of claiming one-by-one with /prewards.
---
### Detailed explanation of what it should do, when and in what situation: ###
Command to add example:
"/prewards claim all"

### Basic explanation as to why this feature is needed and/or useful to have: ###
To allow players to claim their playtime reward one go without having to click and spam one by one. For example: 
![unknown](https://user-images.githubusercontent.com/51330952/83777973-a75a5900-a6bc-11ea-9cca-b8ab5e4b14d6.png)

**Comment by CloudeLecaw** [comment]
A nice udea if you have a sever with may playtime rewards and the player just wants to get all quicky. 
A new command would not be nessesarry
It would be enough to have a new - getall variable
So something like:
**Cmi prewards (playername) (-getall)**

**Comment by BboyZeroOne** [comment]
Bump. I hope this suggestion won't go forgotten in the sea of feature requests/bug reports.
